closure-deps/bin/closuremakedeps.js

```diff
@@ -215,7 +226,7 @@ async function main(opt_args) {
 
   const sources = new Set((args.file || []).map(resolve));
   const roots = (args.root || []).map(resolve);
-  const excluded = new Set((args.exclude || []).map(resolve));
```

Comment:
Doesn't this still make sense to be treated as a set? e.g. if someone specifies `--exclude=path/to/*_test.js --exclude=path/to/*_test.js` we should only process that glob once?